### PR TITLE
Add nc2mmd API usage example

### LINKED ISSUE #107: Document nc2mmd api

### PR DESCRIPTION
**Pull request by mortenwh**
Closes #107

### CODE REVIEW

**Comment by mortenwh** [comment]
@Teddy-1000 - jeg tror denne er klar nå. Hvis du får fikset parsing av int og sjekket noen flere datasett, så bør det være godt nok. Så oppdaterer vi etterhvert som ting evt dukker opp.

File: source/register_context.adoc [hunk 1]
```diff
@@ -35,11 +35,50 @@ These steps are described in the following.
...
+2. Create a json copy of the metadata with the `ncheader2json` script, and push this to the nc2mmd API function:
++
+[source]
+----
+ncheader2json -i ./lebesby_karlmyhr_max_air_temperature.nc | curl -X POST -H 'Content-Type: application/json' -d @- "https://py-mmd-tools.s-enda.k8s.met.no/nc2mmd?return_xml=1&url=https://thredds.met.no/thredds/dodsC/met.no/observations/surface/96220/216/lebesby_karlmyhr_max_air_temperature.nc&parent=no.met:9fbe6a04-279c-46b8-a582-bb08f4b11794" > my_mmd.xml
+----
+
+The API is currently an early version. If it fails, and the error is unclear,
+please try to use the nc2mmd script locally in order to understand what is
+wrong.
```
Comment (Teddy-1000) [inline source/register_context.adoc:64]: This is not entirely true, as long as the flag return_xml is set to false(default) the nc2mmd errors and warnings will also be returned, if only warnings was raised, the resulting mmd is also returned.

**Review by Teddy-1000 — APPROVED** [review]

File: source/register_context.adoc (hunk 1 above)
Comment (mortenwh) [inline source/register_context.adoc:64]:
```suggestion
please contact the Data Manager or try to use the nc2mmd script locally.
```

File: source/register_context.adoc [hunk 2]
```diff
@@ -35,11 +35,50 @@ These steps are described in the following.
...
+
+The OpenAPI documentation for the API can be found at py-mmd-tools.s-enda.k8s.met.no/docs
```
Comment (mortenwh) [inline source/register_context.adoc:66]:
```suggestion
The OpenAPI documentation for the API can be found at https://py-mmd-tools.s-enda.k8s.met.no/docs.
```

File: source/register_context.adoc (hunk 1 above)
Comment (Teddy-1000) [reply source/register_context.adoc]: I think it should say that the API will return an error message if there are errors in nc file header, and if they can not interpret the error message from the API, contact the Data Manager or try the nc2mmd script.

File: source/register_context.adoc [hunk 3]
```diff
@@ -35,11 +35,49 @@ These steps are described in the following.
...
+----
+2. Create a json copy of the metadata with the `ncheader2json` script, and push this to the nc2mmd API function:
++
+[source]
+----
+ncheader2json -i ./lebesby_karlmyhr_max_air_temperature.nc | curl -X POST -H 'Content-Type: application/json' -d @- "https://py-mmd-tools.s-enda.k8s.met.no/nc2mmd?return_xml=1&url=https://thredds.met.no/thredds/dodsC/met.no/observations/surface/96220/216/lebesby_karlmyhr_max_air_temperature.nc&parent=no.met:9fbe6a04-279c-46b8-a582-bb08f4b11794" > my_mmd.xml
+----
+
+The API is currently an early version. If it fails, and the error is unclear,
+please contact the Data Manager or try to use the nc2mmd script locally.
```
Comment (mortenwh) [inline source/register_context.adoc:63]:
```suggestion
please try to use the nc2mmd script locally or contact the Data Manager.
```

**Pull request merged**